frontend/src/layout/navigation-3000/sidepanel/sidePanelLogic.tsx

```diff
@@ -80,7 +81,11 @@ export const sidePanelLogic = kea<sidePanelLogicType>([
                         return true
                     }
 
-                    if (tab === SidePanelTab.Activity && unreadCount) {
+                    if (
+                        tab === SidePanelTab.Activity &&
+                        unreadCount &&
+                        userLogic.values.hasAvailableFeature(AvailableFeature.AUDIT_LOGS)
```

Comment:
this type of access to a logic always feels super bug-prone. Any reason why not to `connect` it?